app/assets/javascripts/member-facing.js.erb

```diff
@@ -19,6 +19,8 @@
 require("<%= Settings.external_js_path %>");
 <% end %>
 
+window.URI = require('urijs');
```

Comment:
I agree that this adds to the leftpad.js problem, but on the other hand I do feel this is the kind of functionality that should be provided by a library. Adding a param to a URL involves some parsing of the URL. I would need to figure out first if there are already other params in there.

Anyway, given there's consensus here about removing this, I'll rewrite the code not to use this package.